application/utils.py

```
@@ -18,21 +18,27 @@
+import logging
 import os
 import smtplib
 import socket
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
 
+logger = logging.getLogger(__name__)
+
+
 def int_to_bytes(num):
```

Ruff

Missing return type annotation for public function `int_to_bytes`

(ANN201)